digdag-core/src/main/java/io/digdag/core/workflow/WorkflowExecutor.java

```diff
@@ -1330,7 +1330,11 @@ private boolean taskSucceeded(TaskControl lockedTask,
         }
         catch (ConfigException ex) {
             // Subtask config or _check task config has a problem (e.g. more than one operator).
-            return taskFailed(lockedTask, buildExceptionErrorConfig(ex).toConfig(cf));
+            Config errorConfig = buildExceptionErrorConfig(ex).toConfig(cf);
+            logger.error("Configuration error at task {}: {}",
```

Comment:
Looks good.

But I'm also wondering if we'd better change the log level of https://github.com/treasure-data/digdag/blob/cefde0d80a6f2a573ac800454da8b4fa7f3e87ad/digdag-core/src/main/java/io/digdag/core/workflow/WorkflowExecutor.java#L1251 to `warn` or `error`